cmd/autoheal/object_template.go

```diff
@@ -118,6 +118,11 @@ type ObjectTemplate struct {
 // parameter.
 //
 func (t *ObjectTemplate) Process(object interface{}, data interface{}) error {
+	if reflect.ValueOf(object).Kind() != reflect.Ptr {
```

Comment:
As you are going to use the `kind` twice, maybe is worth using a local variable:

```go
kind := reflect.ValueOf(object).Kind()
if kind != reflect.Ptr {
  return fmt.Errorf("....", kind)
}
```